app/pathogen/arbovirus/dashboard/(table)/columns.tsx

```diff
@@ -57,6 +57,7 @@ export const columns: DataTableColumnDef<Estimate, unknown>[] = [
   {
     accessorKey: "estimateId",
     header: get_header("Estimate ID"),
+    enableHiding: false,
     cell: ({ row }) => {
```

Comment:
makes sure we never end up with an empty table if all the rows are unselected. Estimate id is the identifier and should not be allowed to be deselected